src/index.html

```diff
@@ -12,108 +12,123 @@
     <script src="system.config.js" type="text/javascript"></script>
-
+    <link href="/node_modules/@syncfusion/ej2-base/styles/material.css" rel="stylesheet" />
+    <link href="/node_modules/@syncfusion/ej2-navigations/styles/appbar/material.css" rel="stylesheet" />
+    <link href="/node_modules/@syncfusion/ej2-layouts/styles/avatar/material.css" rel="stylesheet" />
+    <link href="/node_modules/@syncfusion/ej2-layouts/styles/card/material.css" rel="stylesheet" />
+    <link href="/node_modules/@syncfusion/ej2-notifications/styles/skeleton/material.css" rel="stylesheet" />
+    <link href="./styles/styles.css" rel="stylesheet" />
 </head>
 
 <body style=" font-family: 'Roboto', sans-serif;">
     <div id="container" style=" background: #FAFAFA;">
-        <div class="row" id="overall-title" style="cursor: default">
-            <p class="page-title">Health <b>Stat</b></p>
-            <img src="../../icons/Healthstat.png" style="padding-left: 1.5%; margin-top: 10px; float:left;">
-            <img src="../../icons/Profile_.png" style="padding-right: 1.5%; margin-top: 10px; float:right">
-            <p id="sub-title" style=" float: right">Hi, John Smith</p>
+        <!-- Appbar area -->
+        <div id="appbar">
+            <div>
+                <span class="e-avatar custom-avatar e-avatar-circle"><img src="../../icons/Healthstat.png"></span>
+                <p class="appbar-title"> Health <b>Stat</b> </p>
+                <div class="e-appbar-spacer"></div>
+                <p class="appbar-subtitle"> Hi, John Smith </p>
+                <span class="e-avatar custom-avatar e-avatar-circle"><img src="../../icons/Profile_.png"></span>
+            </div>
         </div>
+        <!-- Card area -->
         <div id="card-row">
             <div class="row">
                 <div class="col-xs-6 col-xl-3 col-lg-3 col-md-3 col-sm-6 calories-valueactive" id="calories-column"
-                    style="padding-right: 0px; padding-left: 20px">
-                    <div class="card" id="calories" style="max-width: 445px; background-image: linear-gradient(45deg, #F23B3B 0%, #EF9027 100%)">
-                        <div class="card-body expense-card" id="calories-bg" style=" background-image: linear-gradient(45deg, #F23B3B 0%, #EF9027 100%); border-radius: 4px">
-                            <div class="card-block" id="caloriescard" style="box-shadow: 0 3px 6px 2px rgba(178,30,195,0.30);">
-                                <div class="media">
-                                    <div class="media-body text-xs-left float-right" id="calories-value" style="max-width: 445px; background-image: linear-gradient(45deg, #F23B3B 0%, #EF9027 100%); border-radius: 4px">
-                                        <h3 class="card-value" id="calories-text" style="color: #FFFFFF">13,100</h3>
+                    style="padding-right: 0px; padding-left: 20px; margin-bottom: 20px;">
```

Comment:
confirm the usage for margin-bottom

Reply:
added an helper class and moved the margin styles to external sheet instead of inline.